applications_tests/lethe-particles/periodic_boundary_box.prm

```diff
@@ -93,9 +93,14 @@ end
 #---------------------------------------------------
 
 subsection DEM boundary conditions
-  set number of boundary conditions = 1
+  set number of boundary conditions = 2
 
-  subsection boundary condition 0
+   subsection boundary condition 0
+    # Choices are fixed_wall|outlet|rotational|translational|periodic
```

Comment:
What is fixed_wall actually? 
Also, maybe (in another pr) we should have a test also for rotational/translational and PBC (like a rotating drum with PBC on flat walls) and make sure all PB match well.